app/helpers/skatepark_helper.rb

```diff
@@ -36,4 +28,8 @@ def num_half_stars(rating)
   def num_stars(rating)
     rating.to_f.floor
   end
+
+  def state_abbrevs
```

Comment:
should this be private?

Reply:
I originally made it private but we use it in `users/show`